CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 # Release Date
 
 ## Release Version
-
+- Adding External Libraries used by MoEngage SDK
```

Comment:
can you double-check the versions in the table? I think 1-2 versions have a typo.